extensions/ql-vscode/src/vscode-tests/run-integration-tests.ts

```diff
@@ -120,7 +122,10 @@ function getLaunchArgs(dir: TestDir) {
       ];
 
     case TestDir.CliIntegration:
-      break;
+      return [
+        path.resolve(__dirname, '../../test/data'),
+        process.env.TEST_CODEQL_PATH!
```

Comment:
This is a list of workspace folders right?

Reply:
Yes. I will add a comment here.